lighthouse-core/report/html/renderer/report-renderer.js

```diff
@@ -253,6 +258,11 @@ class ReportRenderer {
     reportContainer.appendChild(reportSection);
     reportSection.appendChild(this._renderReportFooter(report));
 
+    if (fullPageScreenshot) {
+      reportContainer.appendChild(
+        ElementScreenshotRenderer.createBackgroundImageStyle(this._dom, fullPageScreenshot));
```

Comment:
it might be worth a comment too, since responsibility for initializing this is so far from where the screenshot elements are being created in `details-renderer.js`.